libs/cardano-ledger-conformance/cardano-ledger-conformance.cabal

```
@@ -79,7 +79,7 @@ library
     cardano-ledger-core:testlib,
     cardano-ledger-executable-spec,
     cardano-ledger-shelley:{cardano-ledger-shelley, testlib},
-    cardano-ledger-shelley-test,
+    cardano-ledger-shelley-test >=1.6,
```

You don't need bounds in `cardano-ledger-conformance`, since it is not a package that gets released, so it will alwasy use the version that is in this repo.
```suggestion
    cardano-ledger-shelley-test,
```